aspnetcore/mvc/views/view-components.md

```diff
@@ -82,34 +82,34 @@ We recommend you name the view file *Default.cshtml* and use the *Views/Shared/C
 
 ## Invoking a view component
 
-To use the view component, call `@Component.InvokeAsync("Name of view component", <anonymous type containing parameters>)` from a view. The parameters will be passed to the `InvokeAsync` method.  The `PriorityList` view component developed in the article is invoked from the *Views/Todo/Index.cshtml* view file. In the following, the `InvokeAsync` method is called with two parameters:
+To use the view component, call `@Component.InvokeAsync("Name of view component", <anonymous type containing parameters>)` from a view. The parameters will be passed to the `InvokeAsync` method. The `PriorityList` view component developed in the article is invoked from the *Views/Todo/Index.cshtml* view file. In the following, the `InvokeAsync` method is called with two parameters:
 
 [!code-html[Main](view-components/sample/ViewCompFinal/Views/Todo/IndexFinal.cshtml?range=35)]
 
-## Invoking a view component as a Tag Helper
+## Invoking a view component as a tag helper
```

Comment:
I lost this battle with the PU. They insist on UC Tag Helper in lieu of the correct tag help lowercase. It's out of my hands